evm-ext/src/native_impl.rs

```diff
@@ -121,79 +129,60 @@ fn evm_transact_inner(
         evm_native_ctx.storage_trie,
         evm_native_ctx.resolver,
     ));
-    // todo: storage trie repository factory?
```

Comment:
I guess this was pre-EVM storage trie refactor stuff and is now obsoleted?

Reply:
Yeah, @RomanHodulak would have put that todo there during the EVM trie work, but I didn't see why it was still relevant so it seemed safe to delete. Correct me if I'm wrong Roman!